SAPHana/bin/SAPHanaSR-manageAttr

```diff
@@ -318,16 +328,14 @@ chk_presence_of_file() {
                 chk=1
            fi
         fi
-    done
+    done < <($runCmd "if [ -f $file ]; then echo 'ok'; else echo 'nok'; fi")
```

Comment:
Here I am a bit confused. The echo "ok" seams not to match the if [[ ...line =~  ^OK: 
The colon (:) is missing

Reply:
The 'crm cluster run' returns for every node a line 'OK: [nodename]' or 'ERROR: [nodename]' and then the result of the command to be executed. In this case my simple command prints 'ok', if the file I'm searching for exists and 'nok', if the file is not available.

Reply:
Sorry we should discuss that online (still do not understand it).